npm/vue/test-tsd/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "noEmit": true,
+    "skipLibCheck": true,
+  },
+  "exclude": [],
+  "include": [
+    "**/*"
+  ]
```

Comment:
While selecting entry files, `tsc` processes `includes` first. The `**/*` pater will select all `.ts` files in the project directory. It does not select `.js`, `.d.ts`, or `.vue` extensions. So the select entry points are: `mount-test.ts` and `test.ts`.

Hence there is no need to exclude anything. `"exclude": []` simply overwrites the excludes of parent `tsconfig.json`.